tests/monotouch-test/Foundation/DateTest.cs

```diff
@@ -28,33 +28,32 @@ public class DateTest {
 		public void InLimits ()
 		{
 			// .NET can represent this date just fine
-			Assert.AreEqual (new DateTime (4001, 01, 01), (DateTime)NSDate.DistantFuture, "distant future");
+			Assert.AreEqual (new DateTime (4001, 01, 01), (DateTime) NSDate.DistantFuture, "distant future");
```

Comment:
try to avoid whitespace changes - even if the current code does not follow the code style
makes it easier to see what's the new code/tests vs what's changed vs what's new

Reply:
Let me know if you need help reverting a part of a diff with git. It's actually really cool once you get used to it for stuff like this.